src/features/warpCore/warpCoreConfig.ts

```diff
@@ -0,0 +1,75 @@
+import { warpRouteConfigs as registryWarpRoutes } from '@hyperlane-xyz/registry';
+import { WarpCoreConfig, WarpCoreConfigSchema, validateZodResult } from '@hyperlane-xyz/sdk';
+import { objFilter, objMerge } from '@hyperlane-xyz/utils';
+import { warpRouteWhitelist } from '../../consts/warpRouteWhitelist.ts';
+import { warpRouteConfigs as tsWarpRoutes } from '../../consts/warpRoutes.ts';
+import yamlWarpRoutes from '../../consts/warpRoutes.yaml';
+
+export function assembleWarpCoreConfig(storeOverrides: WarpCoreConfig[]): WarpCoreConfig {
```

Comment:
This is not actually all new code. I just moved it to a new warpCore folder and improved the organization a bit